src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+@font-face {
+  src: url(../src/fonts/Avenir.ttc);
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  width: 100%;
+  font-weight: 300;
+  line-height: 21.86px;
+}
+
+.search-field {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  background-image: url(../src/images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.search--lange {
```

Comment:
The class name `.search--lange` appears to be a typo. It should likely be `.search--large` to match the intended styling for large input fields. Please verify and update the class name accordingly.